CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+
+#### announcer
+
+Generates email diffs summaries to announce product releases.
+
+* Sources: [factory-package-news/announcer.py](factory-package-news/announcer.py)
+* Documentation: [factory-package-news/README.asciidoc](factory-package-news/README.asciidoc)
+* Package: openSUSE-release-tools-announcer
+
+#### pkglistgen
+
+Generates and updates OBS products for openSUSE and SLE. It generates package lists based on
+`000package-groups` and puts them in `000product` (resulting kiwi files) and `000release-packages`
+(release package spec files).
+
+* Sources: [pkglistgen.py](pkglistgen.py)
+* Documentation: [docs/pkglistgen.md](docs/pkglistgen.md)
+* Package: openSUSE-release-tools-pkglistgen
+* Usage: gocd
+
+#### container_cleaner
+
+Cleans up the `maintenance_release` projects for containers
+
+* Sources: [container_cleaner.py](container_cleaner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### metrics
+
+Generates insightful metrics from relevant OBS and annotation data, based on InfluxDB and Grafana.
+
+* Sources: [metrics.py](metrics.py)
+* Documentation: [docs/metrics.md](./docs/metrics.md)
+* Package: openSUSE-release-tools-metrics
+* Usage: ?
+
+#### metrics-access
+
+Ingests `download.opensuse.org` Apache access logs and generate metrics. It is composed of a PHP.
+script and a set of [systemd units](systemd).
+
+* Sources: [metrics/access/aggregate.php](metrics/access/aggregate.php)
+* Documentation: [docs/metrics.md](./docs/metrics.md)
+* Package: openSUSE-release-tools-metrics-access
+* Usage: ?
+
+#### totest-manager
+
+Releases distribution snapshots to openQA and publishes if the result is positive.
+
+* Sources: [totest-manager.py](totest-manager.py) and [ttm](ttm)
+* Documentation: [ttm/README.md](ttm/README.md)
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### unmaintained
+
+Finds unmaintained binaries sourced from SLE.
+
+* Sources: [unmaintained.py](unmaintained.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### sync-rebuild
+
+Syncs openSUSE:Factory and openSUSE:Factory:Rebuild.
+
+* Sources: [sync-rebuild.py](sync-rebuild.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bugowner
+
+Manages bugowner information
+
+* Sources: [bugowner.py](bugowner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bs_mirrorfull
+
+Mirrors repositories from build service.
+
+* Souces: [bs_mirrorfull](bs_mirrorfull)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: Used by other tools like `pkglistgen` or `repocheck`
+
+#### biarchtool
+
+Manages biarch packages
+
+* Source: [biarchtool.py](biarchtool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### build-fail-reminder
+
+Sends e-mails about packages failing to build for a long time.
+
+* Sources: [build-fail-reminder.py](build-fail-reminder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
```

Comment:
and I think transitive it is send to opensuse-factory mailing  list and maybe also to individual bug owners.

Reply:
Maybe a link to gocd/build-fail-reminder.gocd.yaml ?

BTW, at this point of the review is pretty clear to me that the list of tools is pretty much unsorted. What about something like having all the tools deployed at gocd at the start and all the obsolete ones at the end?